lib/Event.php

```diff
@@ -23,96 +23,128 @@ class Event extends ApiResource
+    const CUSTOMER_SUBSCRIPTION_CREATED            = 'customer.subscription.created';
+    const CUSTOMER_SUBSCRIPTION_DELETED            = 'customer.subscription.deleted';
+    const CUSTOMER_SUBSCRIPTION_TRIAL_WILL_END     = 'customer.subscription.trial_will_end';
+    const CUSTOMER_SUBSCRIPTION_UPDATED            = 'customer.subscription.updated';
+    const FILE_CREATED                             = 'file.created';
+    const INVOICE_CREATED                          = 'invoice.created';
+    const INVOICE_DELETED                          = 'invoice.deleted';
+    const INVOICE_MARKED_UNCOLLECTIBLE             = 'invoice.marked_uncollectible';
+    const INVOICE_PAYMENT_FAILED                   = 'invoice.payment_failed';
+    const INVOICE_PAYMENT_SUCCEEDED                = 'invoice.payment_succeeded';
+    const INVOICE_SENT                             = 'invoice.sent';
+    const INVOICE_UPCOMING                         = 'invoice.upcoming';
+    const INVOICE_UPDATED                          = 'invoice.updated';
+    const INVOICE_VOIDED                           = 'invoice.voided';
+    const INVOICEITEM_CREATED                      = 'invoiceitem.created';
+    const INVOICEITEM_DELETED                      = 'invoiceitem.deleted';
+    const INVOICEITEM_UPDATED                      = 'invoiceitem.updated';
+    const ISSUER_FRAUD_RECORD_CREATED              = 'issuer_fraud_record.created';
+    const ISSUING_AUTHORIZATION_CREATED            = 'issuing_authorization.created';
+    const ISSUING_AUTHORIZATION_UPDATED            = 'issuing_authorization.updated';
+    const ISSUING_CARD_CREATED                     = 'issuing_card.created';
+    const ISSUING_CARD_UPDATED                     = 'issuing_card.updated';
+    const ISSUING_CARDHOLDER_CREATED               = 'issuing_cardholder.created';
+    const ISSUING_CARDHOLDER_UPDATED               = 'issuing_cardholder.updated';
+    const ISSUING_DISPUTE_CREATED                  = 'issuing_dispute.created';
+    const ISSUING_DISPUTE_UPDATED                  = 'issuing_dispute.updated';
+    const ISSUING_TRANSACTION_CREATED              = 'issuing_transaction.created';
+    const ISSUING_TRANSACTION_UPDATED              = 'issuing_transaction.updated';
+    const ORDER_CREATED                            = 'order.created';
+    const ORDER_PAYMENT_FAILED                     = 'order.payment_failed';
+    const ORDER_PAYMENT_SUCCEEDED                  = 'order.payment_succeeded';
+    const ORDER_UPDATED                            = 'order.updated';
+    const ORDER_RETURN_CREATED                     = 'order_return.created';
+    const PAYMENT_INTENT_AMOUNT_CAPTURABLE_UPDATED = 'payment_intent.amount_capturable_updated';
+    const PAYMENT_INTENT_CREATED                   = 'payment_intent.created';
+    const PAYMENT_INTENT_PAYMENT_FAILED            = 'payment_intent.payment_failed';
+    const PAYMENT_INTENT_REQUIRES_CAPTURE          = 'payment_intent.requires_capture';
```

Comment:
Can you remove this one? We're actively deprecating it.

Reply:
Roger.